runtime/vm/JFRConstantPoolTypes.hpp

```diff
@@ -1626,12 +1626,12 @@ class VM_JFRConstantPoolTypes {
 				}
 			}
 			entry->classLoaderData = (U_64)classLoader;
-			entry->classCount = hashTableGetCount(classLoader->classHashTable);
+			entry->classCount = classLoader->loadedClassCount;
```

Comment:
if this classloader is the anonClassLoader use `_javaVM->anonClassCount` instead